### PR TITLE
added goodbye

### PR DESCRIPTION
**Pull request by JISock**
I added print("goodbye")

### CODE REVIEW

File: file.py [hunk 1]
```diff
@@ -1 +1,2 @@
 print("hello jonny")
+print("goodbye world")
```
Comment (jxz12) [inline file.py]: this should say goodbye jonny instead

**Review by jxz12 — CHANGES_REQUESTED** [review]
say goodbye to jonny

File: file.py (hunk 1 above)
Comment (pamelahathway) [inline file.py]: please add a new line at the end of the the file

File: file.py [hunk 2]
```diff
@@ -1 +1,2 @@
 print("hello jonny")
```
Comment (pamelahathway) [inline file.py]: Please capitalize Jonny (he forgot to do that)

**Review by pamelahathway — CHANGES_REQUESTED** [review]
Well done, few small changes, then I'm happy to approve :).

File: file.py (hunk 1 above)
Comment (JISock) [reply file.py]: Done

File: file.py (hunk 2 above)
Comment (JISock) [reply file.py]: Done

**Review by jxz12 — APPROVED** [review]
kool

**Review by pamelahathway — APPROVED** [review]

**Pull request merged**